test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -23,7 +23,8 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
         pytest.skip("Currently skipping for TF2.3.0 on p2.8xlarge until the issue is fixed")
```

Comment:
Please remove this skip for p2.8xlarge.

Reply:
This is for 2.3.0, will be removed in the PR below

https://github.com/aws/deep-learning-containers/pull/623/files